lib/full_text_search/issue_content_synchronizable.rb

```diff
@@ -0,0 +1,38 @@
+module FullTextSearch
+  module IssueContentSynchronizable
+    extend ActiveSupport::Concern
+
+    included do
+      after_commit :queue_sync_on_commit
+      after_destroy :queue_sync_on_destroy
+    end
+
+    private
+
+    def queue_sync(action, options = {})
```

Comment:
Is "query" better name for this...? In general, "query" means `SELECT` SQL in Rails application context...

Reply:
fix: [bee54aa](https://github.com/clear-code/redmine_full_text_search/pull/167/commits/bee54aae1f30b1ab10b7749bd53e7a59766d6374) Ahhh I see. I renamed it to `term`.

Reply:
Ah, sorry. I thought "queue_sync" includes "query" not "queue"...